app/__init__.py

```diff
@@ -149,15 +149,16 @@ def create_app():
         from app.api.admin_translations import admin_blueprint
         from app.api.orders import alipay_blueprint
         from app.api.settings import admin_misc_routes
+        from app.api.server_version import info_route
 
         app.register_blueprint(api_v1)
         app.register_blueprint(event_copy)
         app.register_blueprint(upload_routes)
         app.register_blueprint(export_routes)
-        app.register_blueprint(import_routes)
         app.register_blueprint(celery_routes)
         app.register_blueprint(auth_routes)
         app.register_blueprint(event_statistics)
+        app.register_blueprint(info_route)
```

Comment:
Still at the wrong place

Reply:
I have moved the `/info_route` and `/import_route` to its original place. Do you want me to put `/info_route` end?

Reply:
Yes

Reply:
info_route had no original place as it's a new route and should be placed in the end